src/components/Observations/index.tsx

```diff
@@ -57,5 +62,23 @@ export default function Observations(): JSX.Element {
     return <CircularProgress sx={{ margin: 'auto' }} />;
   }
 
-  return <ObservationsHome />;
+  return (
+    <Switch>
+      <Route exact path={APP_PATHS.OBSERVATION_MONITORING_PLOT_DETAILS}>
+        <ObservationMonitoringPlotDetails />
+      </Route>
+      <Route exact path={APP_PATHS.OBSERVATION_PLANTING_ZONE_DETAILS}>
+        <ObservationPlantingZoneDetails />
+      </Route>
+      <Route exact path={APP_PATHS.OBSERVATION_DETAILS}>
+        <ObservationDetails />
+      </Route>
+      <Route exact path={APP_PATHS.PLANTING_SITE_OBSERVATIONS}>
+        <ObservationsHome />
+      </Route>
```

Comment:
Is this route needed, given that the next `'*'` route is the same?

Reply:
I should have explained clearly.
Analogy is a switch statement:
```
  switch(arg) {
     case xyz: <code>;
     case def: <code>;
     case rst:  // here both rst and default have same return value, we could skip mentioning rst but it is good practice to express this intent
     default:
           <code>;
  }
```

Reply:
Thanks